packages/mermaid/src/diagrams/gantt/parser/gantt.jison

```diff
@@ -79,6 +79,7 @@ that id.
 
 "gantt"                     return 'gantt';
 "dateFormat"\s[^#\n;]+      return 'dateFormat';
+"compact"                   return 'compact';
```

Comment:
An issue with the compact keyword.

Reply:
Moved to different approach (see comment)